configure

```diff
@@ -88,11 +87,10 @@ struct_passing=""
 struct_return=""
 
 case "$target" in
-  powerpc-*|ppc-*|powerpc64-*|ppc64-*|e5500-*)
```

Comment:
Why not keeping the possibility to configure for e5500?  I mean, this would set "e5500" as the default value for the -mcpu flag.